examples/src/socketio-retry/socketio-retry.rs

```diff
@@ -0,0 +1,82 @@
+//! This a end to end test server used with this [test suite](https://github.com/socketio/socket.io-protocol)
+
+use std::time::Duration;
+
+use hyper::Server;
+use serde_json::Value;
+use socketioxide::errors::{AckSenderError, BroadcastError, SendError, TransportError};
+use socketioxide::{Namespace, SocketIoConfig, SocketIoService};
+use tokio::time::sleep;
+use tracing::{info, Level};
+use tracing_subscriber::FmtSubscriber;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let subscriber = FmtSubscriber::builder()
+        .with_line_number(true)
+        .with_max_level(Level::INFO)
+        .finish();
+    tracing::subscriber::set_global_default(subscriber)?;
+
+    let config = SocketIoConfig::builder()
+        .ping_interval(Duration::from_millis(300))
+        .ping_timeout(Duration::from_millis(200))
+        .max_payload(1e6 as u64)
```

Comment:
These weird configs are used only for the test suites and shouldn't be in the examples (that's why I moved e2e testing binaries from the examples)

Reply:
Not sure that it should be in examples at all, it's not mandatory to handle this error